### PR TITLE
Containerize Dashboard  to deploy on Review app eks cluster

### PR DESCRIPTION
**Pull request by dskgsa**
### Containerized Dashboard  so that we can deploy it into our review apps.
https://gitlab.login.gov/lg-people/platform/mary/team-mary/-/issues/9

### Description of Changes:
Add Dockerfile
Add gitlab CI yml to support gitlab pipelines
Updated Config files to support docker contianer

### CODE REVIEW

**Review by timothy-spencer — APPROVED** [review]
It would be great if that saml_sp_private_key entry was marked with some sort of "this is a dummy key, not used by prod" comment.  But otherwise, looks good!

**Comment by timothy-spencer** [comment]
Oh, and @mitchellhenke should approve this too, to make sure that it meets his requirements.

File: config/oidc.yml.docker [hunk 1]
```diff
@@ -0,0 +1,10 @@
+test: &default
```
Comment (mitchellhenke) [inline config/oidc.yml.docker:1]: this file looks to have the same content as `config/oidc.yml`, could we use that instead?

**Comment by dskgsa** [comment]
Success full Pipeline
https://gitlab.login.gov/lg/identity-dashboard/-/pipelines/95949

**Comment by dskgsa** [comment]
@mitchellhenke  updated as per the comments.
 latest pipeline 
https://gitlab.login.gov/lg/identity-dashboard/-/pipelines/95963

**Review by mitchellhenke — APPROVED** [review]

**Pull request merged**